### ISSUE TITLE
The quantization example , mnist_cnn.py  is wrong,array softmax/Softmax, is lacking min/max data, which is necessary for quantization

### ISSUE THREAD

**Issue by ptklx** [tensorflow/model-optimization#108]
the code

2019-09-20 17:04:32.110476: I tensorflow/lite/toco/graph_transformations/graph_transformations.cc:39] Before Group bidirectional sequence lstm/rnn: 8 operators, 17 arrays (1 quantized)
2019-09-20 17:04:32.110534: F tensorflow/lite/toco/tooling_util.cc:1709] Array quantize_emulate_wrapper_3/BiasAdd, which is an input to the Softmax operator producing the output **array softmax/Softmax, is lacking** min/max data, which is necessary for quantization. If accuracy matters, either target a non-quantized output format, or run quantized training with your model from a floating point checkpoint to change the input graph to contain min/max information. If you don't care about accuracy, you can pass --default_ranges_min= and --default_ranges_max= for easy experimentation.
Fatal Python error: Aborted

the 
https://github.com/tensorflow/model-optimization/blob/master/tensorflow_model_optimization/python/examples/quantization/keras/mnist_cnn.py

**Comment by alanchiao** [comment]
Hi Dannis,

Thanks for reporting the issue. Notably, the quantization aware training project is a work-in-progress (as you can tell from the commits and our [roadmap](https://www.tensorflow.org/model_optimization/guide/roadmap), including the example you looked at.

We'll make sure that all such examples are fixed prior to launch but closing this issue since it's intended right now (we haven't gotten to it).

**Comment by mpariente** [comment]
As there has been a new commit to the mentionned file since then, is this supposed to be fixed? 
I don't get the same error message but pretty close. 
Any news? 

Thanks

**Comment by alanchiao** [comment]
@mpariente : the training-time quantization library is not intended for usage yet.

You can try Dannis' fix in the meantime, but there is no guarantee that another change will not cause a breakage as we focus on the making the library ready.

**Comment by mpariente** [comment]
Ok, good to know. 
Thanks for you answer !
